### ISSUE TITLE
Unsupported argument "hosted_zone_name" for module "bastion"

### ISSUE THREAD

**Issue by jross-ai** [Guimove/terraform-aws-bastion#71]
I am getting an odd issue where my terraform plan step is now failing due to the unsupported argument "hosted_zone_name" in the "bastion" module. The below plan previously worked. It seems that "hosted_zone_name" has been replaced with "hosted_zone_id". However, I have not found any documentation indicating this change. Moreover, the [terraform registry page](https://registry.terraform.io/modules/Guimove/bastion/aws/1.2.0?tab=inputs) for this repo shows that "hosted_zone_name" is still a valid input. It seems like this repo though no longer lists "hosted_zone_name" as a variable. Is there answer that the "hosted_zone_name" variable should now be "hosted_zone_id"? Thank you!

Code:
```
module "bastion" {
  source = "git::https://github.com/Guimove/terraform-aws-bastion.git?ref=master"
  bucket_name = module.bastion_logs_s3.bucket_id
  region = var.region
  vpc_id = module.vpc.vpc_id
  is_lb_private = "false"
  bastion_host_key_pair = var.stage
  hosted_zone_name = var.hosted_zone_id
  bastion_record_name = local.bastion_hostname
  elb_subnets = module.subnets.public_subnet_ids
  auto_scaling_group_subnets = module.subnets.public_subnet_ids
  create_dns_record = "true"
  tags = var.tags
}
```

Error:
<img width="546" alt="Screen Shot 2020-03-19 at 7 49 35 PM" src="https://user-images.githubusercontent.com/53832644/77124980-cbda5980-6a1a-11ea-8bf0-db32e586968b.png">

**Comment by billwanjohi** [comment]
You are correct that the new variable name is `hosted_zone_id`, which is more accurate, but still has a poor description. The change happened in https://github.com/Guimove/terraform-aws-bastion/pull/65/commits/7f3e6c27426ed645df2b8c896f5eaf3d0c05b423, which was included in tag 1.20.1.

For people suprised by this new behavior, you can either pin yourself to an older version

```terraform
source = "Guimove/bastion/aws"
version = "1.2.0"
hosted_zone_name = ...
``` 

or bump the required version and rename the variable 

```terraform
source = "Guimove/bastion/aws"
version = "~> 1.2.1"
hosted_zone_id = ...
```

@Guimove if at all possible when you're releasing breaking changes like this in the future, can you [bump the major version](https://www.terraform.io/docs/extend/best-practices/versioning.html)? That way users like ourselves can benefit from non-breaking changes but hold off on any that will require revisions in our code.

(Thanks for putting together this module, it is very handy!)

**Comment by Guimove** [comment]
Hello,

Sorry, I've created a new 2.0.0 version for all these new major changes.
